docs/articles/call-settings.md

```diff
@@ -1,3 +1,37 @@
 # Call settings
 
-(Chris Bacon to write this)
\ No newline at end of file
+## Introduction
+
+All C# [gRPC](http://www.grpc.io/) APIs allow customization of RPC calls by using [CallSettings](../obj/api/Google.Api.Gax.CallSettings.yml). If `CallSettings` are not specified sensible defaults are automatically provided.
+
+There are three distinct places where `CallSettings` may optionally be provided:
+
+* When calling an RPC method *(highest priority)*.
+* Client-wide when creating a gRPC client.
+* Per-RPC-method when creating a gRPC client *(lowest priority)*.
+
+It's important to understand exactly *which* properties of which `CallSettings` will be used during an RPC invocation.
```

Comment:
It may be worth explaining this in terms of "the different settings objects are merged into a single effective `CallSettings`. Each property is merged separately: the highest priority non-null value for each property is used." Or something like that.